scripts/modules/news.js

```diff
@@ -0,0 +1,3 @@
+export function News() {
```

Comment:
I don't see how News is used. Could we delete it?

Reply:
Fixed